package/MDAnalysis/coordinates/PDB.py

```diff
@@ -922,8 +930,12 @@ def write_all_timesteps(self, obj):
 
         will be writing frames 12, 14, 16, ...
 
+
         .. versionchanged:: 0.11.0
            Frames now 0-based instead of 1-based
+
+        .. versionchanged:: 2.0.0
+           CONECT_ record moved to :meth:`close`
```

Comment:
I wonder if we should be a little bit more explicit, maybe in the module nodes, i.e. tell users specifically that CONECT records won't be written until they close the object (would users be likely to use the files before the objects have finished closing? thoughts?)

Reply:
>would users be likely to use the files before the objects have finished closing?

Hopefully not, but I can be more explicit in the docstring for sure.

Reply:
If possible? Maybe just a quick extra line after the bit about writing CONECT records in `notes`? I'll leave it to your discretion.